synapse/lib/scope.py

```diff
@@ -64,8 +64,8 @@ def get(self, name, defval=None):
         Retrieve a value from the closest scope frame.
         '''
         for frame in reversed(self.frames):
-            valu = frame.get(name)
-            if valu != None:
+            valu = frame.get(name, defval)
+            if valu != defval:
```

Comment:
wont this comparison incorrectly fail if the *real* (actually present) value happens to be the same as the default value?  probably best to use the "novalu" thing from synapse/common.py here.